workloads/test.yml

```diff
@@ -67,15 +67,15 @@
       changed_when: false
       register: scale_ci_tooling_ns_exists
 
-    # This delete should block until complete
-    - name: Block for existing tooling namespace
-      block:
-        - name: Ensure any stale scale-ci-workload job is deleted
-          shell: |
-            oc delete job scale-ci-workload -n scale-ci-tooling
-          ignore_errors: true
-      when:
-        - scale_ci_tooling_ns_exists.rc == 0
+    - name: Ensure any stale scale-ci-test job is deleted
+      shell: |
+         oc delete job scale-ci-test -n scale-ci-tooling
```

Comment:
3 spaces.  I didn't point out all of these in the PR, sorry.

Reply:
Ah, no worries I think I got all of them.  Maybe we can get a linter bot/job to find these more easily.

Reply:
Yep, some ci with a linter would be nice.